utils/rowgroup/rowgroup.cpp

```diff
@@ -1043,6 +1043,103 @@ const CHARSET_INFO* Row::getCharset(uint32_t col) const
   return charsets[col];
 }
 
+void Row::compileIsNull(llvm::IRBuilder<>& b, llvm::Value* dataValue, llvm::Value* isNull, uint32_t colIndex)
```

Comment:
I am curious why do you think this functionality belongs here and not to jit directory?

Reply:
The initial idea was to maintain consistency in function call behavior. It seems better to add a tool class in the jit directory to make the JIT code more centralized.

Reply:
> I am curious why do you think this functionality belongs here and not to jit directory?

I have moved it to a compilehelper class.